rdr_service/tools/tool_libs/curation.py

```diff
@@ -372,13 +376,21 @@ def _get_base_src_clean_answers_select(session):
                 QuestionnaireResponseAnswer.valueString.isnot(None)
             ),
             QuestionnaireResponse.status != QuestionnaireResponseStatus.IN_PROGRESS,
-            QuestionnaireResponse.isDuplicate.is_(False)
+            QuestionnaireResponse.isDuplicate.is_(False),
+            or_(
+                ParticipantSummary.dateOfBirth.is_(None),
```

Comment:
Based on a communication to the curation team, we shouldn't be sending through participants that don't have a DOB entered https://precisionmedicineinitiative.atlassian.net/browse/ROC-1034?focusedCommentId=89587. Sorry that didn't make it all the way to the DA 😬

Reply:
Done!